index.html

```diff
@@ -2624,12 +2618,17 @@ <h2>WoT Runtime</h2>
             <p>
                 Technically, the <a>Thing</a> abstraction and its <a>Interaction Model</a> is implemented in a runtime system.
                 This <a>WoT Runtime</a> maintains the execution environment for the behavior implementation and is able to expose and/or consume <a>Things</a>,
```

Comment:
execution context

Reply:
Execution context is mainly used in the context of JavaScript. There it even means something different, as it includes the current evaluation state of code. Will not change.

Reply:
In this PR you "make clear WoT API is only ECMAScript". Then please use the runtime terms defined in the [ECMAScript spec](http://www.ecma-international.org/ecma-262/9.0/index.html#sec-executable-code-and-execution-contexts) and don't assume everyone understands the same thing as you on "Execution Environment".